### PR TITLE
Fix  Yu-Gi-Oh 6 crash with edit card in Android version

### PR DESCRIPTION
**Pull request by sum2012**
This revert part of
https://github.com/hrydgard/ppsspp/commit/74d8a9bdba9b8348e60e8a6b40b01da26b5eb020

### CODE REVIEW

**Comment by hrydgard** [comment]
Okay, will need to figure out what's really wrong later though... as this code makes things clearer when looking at the disassembly in the jit viewer.

**Comment by unknownbrackets** [comment]
Well, more importantly, if this causes a crash, what it means is that the jit is doing something very wrong or that we don't understand.

It's like, if every time you stand up, you get a pain in your hip.  So, you buy a wheelchair and never see the doctor.  Problem solved.  It's not even really about the benefit of the BKPTs... there should not be a pain in your hip hwen you stand up.  Maybe you actually have cancer.

-[Unknown]

**Comment by hrydgard** [comment]
Well, I will figure it out and fix it, this is only temporary so that the rest of the game can get tested by users before 1.0 :)

**Comment by sum2012** [comment]
Just a note:
Ghost In The Shell Stand Alone Complex USA still don't work in Android version with JIT

**Comment by unknownbrackets** [comment]
So, now that v1.0.1 is out do we want to bring this back or do something else here?

Are we not invalidating the icache somewhere?

-[Unknown]

**Comment by sum2012** [comment]
Can we try to port the BKPTs thing to windows to easier to debug in  Android ?

**Comment by hrydgard** [comment]
@unknownbrackets we definitely want to bring this back, but would be good to at least understand what's wrong and preferably fix it before doing so.. It's quite mysterious, as far as I can tell we do invalidate the icache when updating these.

**Comment by sum2012** [comment]
@hrydgard I see.I will try revert this fix when fixed some JIT code.

File: Core/MIPS/JitCommon/JitBlockCache.cpp [hunk 1]
```diff
@@ -409,19 +409,20 @@ void JitBlockCache::LinkBlockExits(int i) {
 			int destinationBlock = GetBlockNumberFromStartAddress(b.exitAddress[e]);
 			if (destinationBlock != -1) 	{
 #if defined(ARM)
-				const u8 *nextExit = b.exitPtrs[e + 1];
-				if (!nextExit) {
-					nextExit = b.normalEntry + b.codeSize;
```
Comment (sum2012) [inline Core/MIPS/JitCommon/JitBlockCache.cpp]: This variable "nextExit " no use anymore ?

**Comment by sum2012** [comment]
Just a record,https://github.com/unknownbrackets/ppsspp/commit/6cc84ffdbb0cf71af6457d9d6eeafd4cc21828f6 don't help this

**Comment by hrydgard** [comment]
@sum2012 , ok, thanks for testing

**Comment by sum2012** [comment]
v1.1.1-62-ge632b32 worse than before,after in story crash
log:
https://gist.github.com/sum2012/3ddb92db0a094e94ae55

**Comment by sum2012** [comment]
curent master also have problem.
I will do git bisect soon

**Comment by sum2012** [comment]
It only happen when first time save
Fast test
last work: v0.9.8
first break v0.9.9

**Pull request merged**